pkg/kubelet/util/pluginwatcher/plugin_watcher.go

```diff
@@ -210,7 +210,12 @@ func (w *Watcher) traversePluginDir(dir string) error {
 	})
 }
 
+const volRootDir = "kubernetes.io"
```

Comment:
To address @vishh concerns about CSI-specific leaks, what if this was renamed to `excludedDirs` which then provides list of directories not to scan for socket.